nebula/ui/components/VPNBottomNavigationBarButton.qml

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import QtQuick.Layouts 1.14
+import Mozilla.VPN 1.0
+
+VPNIconButton {
+    id: root
+
+    property var _screen
+    property alias _source: image.source
+    property bool _hasNotification: true
+
+    accessibleName: ""
+
+    checked: VPNNavigator.screen === _screen
+    onClicked: {
+        if (VPNNavigator.screen === _screen) {
+            window.unwindCurrentStackView();
```

Comment:
Because this needs to be centralized, can we keep it out from this PR?
For instance, we need something similar when the user wants to open the settings from the menubar/systray.

Reply:
done!